src/Controls/src/Core/Span.cs

```diff
@@ -76,6 +76,8 @@ public string Text
 
 		public static readonly BindableProperty FontAttributesProperty = FontElement.FontAttributesProperty;
 
+		public static readonly BindableProperty FontScalingEnabledProperty = FontElement.FontScalingEnableProperty;
```

Comment:
Is this possible? To have some text and only scale a span? On iOS this property is for the label only and might not be possible.

Reply:
All of our spans have a point where a UIFont or textsize is applied so it should work.

We currently don't have Spans implement on Maui though so we'll have to test once Spans are implemented

https://github.com/dotnet/maui/issues/1793